less/home-page.less

```diff
@@ -1,41 +1,86 @@
 .home {
 
+  header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 20px 5px;
+   @media@mobile{
+     display: flex;
+     flex-direction: column;
+     
+   } 
+  }
+    
+    .banner {
+     border-top: 2px dashed grey;
+     
+    }
+
+  .top-content {
+    display: flex;
+    flex-direction: column;
+    border-bottom: 2px dashed grey;
+    padding: 10px 0; 
+    @media@mobile{
+    display: flex;
+    flex-direction: column;
+    }
+  }
+
+
   .content-section {
     margin: 30px 0;
     display: flex;
     flex-wrap: wrap;
     justify-content: space-between;
+    padding: 30px 0;
+    @media@mobile{
+      display: flex;
+    flex-direction: column;
+    // max-width: 100%;
+
+    }
 
    
     
     .text-content {
       width: 48%;
       padding-right: 1%;
+      @media@mobile {
+      width: 100%;
+      }
     }
-
+  
     .img-content {
       width: 48%;
```

Comment:
if you want to match the design file for mobile, be sure to add a breakpoint here and change this width!